src/pages/home.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react';
+
+class Home extends PureComponent {
+  render() {
+    return (
+      <div className="about">
+        <h2>Welcome to our page</h2>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas
+          placeat nam voluptates ab? Mollitia distinctio incidunt iste
+          necessitatibus facilis, sequi repellendus, deserunt officia illo iure
+          minima eligendi impedit eveniet vero.
+        </p>
+        <p>
+          Alias hic quos consequatur, neque odit repudiandae corporis quis
+          aperiam laboriosam in nobis commodi eveniet sint numquam voluptatem
+          rerum ipsam similique, omnis a corrupti laborum natus tempore! Aut,
+          aliquam quibusdam!
+        </p>
+        <p>
+          Quod commodi, quia voluptate at atque quis laboriosam illum architecto
+          quasi veritatis eos magni ut dicta ad, vitae, explicabo quidem. Autem
+          nulla a recusandae similique, eaque aut eum! Ea, magnam.
+        </p>
+        <p>
+          Itaque maiores amet, eum cupiditate exercitationem porro libero
+          consequatur esse optio nostrum sed aliquam fugit repudiandae alias
+          unde minima soluta, excepturi nemo. Maiores nemo quos dolorem at
+          voluptatibus nihil ipsa.
```

Comment:
- [ ] OPTIONAL: I don't think using lorem Ipsum is a good idea here. You can populate it with your own content. :pray: